src/bootstrap/lib.rs

```diff
@@ -605,8 +605,8 @@ impl Build {
         self.verbosity > 0
     }
 
-    pub fn is_very_verbose(&self) -> bool {
-        self.verbosity > 1
+    pub fn cargo_verbosity_level(&self) -> usize {
+        if self.verbosity == 0 { 0 } else { self.verbosity - 1 }
```

Comment:
This can be simply `self.verbosity.saturating_sub(1)`

Alternatively you could use this in `src/bootstrap/builder.rs` to entirely get rid of this function. 

```rust
// if rustbuild is invoked with N `-v`s, we pass N-1 `-v`s to cargo.
for _ in 1..self.verbosity {
    cargo.arg("-v");
}
```